vivisect/base.py

```diff
@@ -216,23 +223,37 @@ def _handleADDSEGMENT(self, einfo):
         self.segments.append(einfo)
 
     def _handleADDRELOC(self, einfo):
-        rva, rtype = einfo
+        if len(einfo) == 2:     # FIXME: legacy: remove after 02/13/2020
```

Comment:
because it's for backwards-compatibility, and visi typically prefers to burn down old .viv's and regen newer.  this gives a year for older vivs to still function as is.

Reply:
:+1: